hack/test-cmd.sh

```diff
@@ -188,6 +210,22 @@ atomic-enterprise start \
   --etcd-dir="${ETCD_DATA_DIR}" \
   --images="${USE_IMAGES}"
 
+os::test::junit::declare_suite_start "cmd/validatation"
+# validate config that was generated
+os::cmd::expect_success_and_text "openshift ex validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml" 'SUCCESS'
+os::cmd::expect_success_and_text "openshift ex validate node-config ${NODE_CONFIG_DIR}/node-config.yaml" 'SUCCESS'
+# breaking the config fails the validation check
+cp ${MASTER_CONFIG_DIR}/master-config.yaml ${BASETMPDIR}/master-config-broken.yaml
+os::util::sed '7,12d' ${BASETMPDIR}/master-config-broken.yaml
+os::cmd::expect_failure_and_text "openshift ex validate master-config ${BASETMPDIR}/master-config-broken.yaml" 'ERROR'
+
+cp ${NODE_CONFIG_DIR}/node-config.yaml ${BASETMPDIR}/node-config-broken.yaml
+os::util::sed '5,10d' ${BASETMPDIR}/node-config-broken.yaml
+os::cmd::expect_failure_and_text "openshift ex validate node-config ${BASETMPDIR}/node-config-broken.yaml" 'ERROR'
+echo "validation: ok"
+os::test::junit::declare_suite_end
```

Comment:
It's an interesting thought. I think before `os::cmd` these types of declarations had value in that it was difficult to tell if forward progress was being made or not. What do we gain by keeping them at all now? Also, I decided to not add suite-specific information to  `os::test::junit:: declare_suite_end` as without a stack that gets hairy on reconciliation.